modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -294,7 +273,7 @@ protected Response uploadNoTransaction(@Context HttpServletRequest request,
                 }
             }
         }
-        return buildResponse(status, result, isMultipart);
+        return buildResponse(status, result, false);
```

Comment:
```suggestion
        return buildResponse(status, result);
```
We may probably. want to deprecate `buildResponse(StatusType, Object, boolean)` and `buildHTMLResponse(StatusType, String)` as they won't be used anymore.

Reply:
Deprecated with the annotation only as these methods are protected.

Reply:
> Deprecated with the annotation only as these methods are protected.

This change doesn't appear currently in the branch, maybe not pushed yet?

Reply:
Yes will be updated in next push